src/SprykerEco/Zed/Payone/Business/PayoneFacadeInterface.php

```diff
@@ -404,15 +425,32 @@ public function isPaymentCapture($idSalesOrder, $idSalesOrderItem);
      *
      * @return \Generated\Shared\Transfer\CheckoutResponseTransfer
      */
+    public function orderPostSave(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponse): CheckoutResponseTransfer;
+
+    /**
```

Comment:
Change the order to:
1. Specification.
2. `@api`.
3. `@deprecated`.

Reply:
++